source/reference/beta-program/namespace-remapping.txt

```diff
@@ -12,11 +12,9 @@ Change Database Names
    :depth: 2
    :class: singlecol
 
-.. include:: /includes/beta-feature
```

Comment:
Can we change this page name to `Namespace Remapping` to match the feature in the release notes?